console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc

```diff
@@ -0,0 +1,55 @@
+= Customise Browser Title
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+The goal of this issue is to provide support in HAL to configure system-wide server side EJB interceptors through the ejb3 subsystem.
```

Comment:
May I have a suggestion? I wonder whether it is possible to implement the following feature:

1. User needs to provide only path to an interceptor .jar - module is created automatically (so explicit EAP configuring and manual copying won't be needed).

Reply:
That's a good suggestion. HAL can only use DMR operations to talk to the management layer. AFAIK there's no way to create a module using DMR.